modules/sample/src/main/resources/issues/issue222.yaml

```diff
@@ -0,0 +1,26 @@
+swagger: '2.0'
+info:
+  title: someapp
+  description: someapp
+  version: '1'
+basePath: "/v1"
+schemes:
+  - http
+produces:
+  - application/json
+paths: {}
+definitions:
+  Request:
+    description: Request fields with id
+    allOf:
+      - "$ref": "#/definitions/RequestFields"
+      - type: object
+        properties:
+          id:
+            type: string
```

Comment:
### This doesn't render both `id` and `id2`
```suggestion
            type: string
      - type: object
        properties:
          id2:
            type: string
```
### This doesn't render parameters from _any_ object literals, so `id` is missing
```suggestion
            type: string
      - "$ref": "#/definitions/RequestFields2"
```